helm/charts/nginx/values.yaml

```diff
@@ -0,0 +1,25 @@
+# Default values for nginx.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: ClusterIP
+  port: 8080
+
+ingress:
+  enabled: true
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - demo.gluu-helm-example.tk
```

Comment:
confine to all our examples. Unify this in all your files to `demoexample.gluu.org`